google-cloud-error_reporting/lib/google/cloud/error_reporting/middleware.rb

```diff
@@ -141,56 +161,74 @@ def report_exception env, exception
         # @param [Hash] env Rack environment hash
         # @param [Exception] exception Exception to convert from
         #
-        # @return [
-        #   Google::Devtools::Clouderrorreporting::V1beta1::ReportedErrorEvent]
-        #   The gRPC ReportedErrorEvent object that's based
-        #   on given exception
+        # @return [Google::Cloud::ErrorReporting::ErrorEvent] The gRPC
+        #   ErrorEvent object that's based on given env and exception
         #
-        def build_error_event_from_exception env, exception
-          error_event = ErrorEvent.from_exception exception
+        def error_event_from_exception env, exception
+          error_event = ErrorReporting::ErrorEvent.from_exception exception
+
+          error_event.user = ENV["USER"]
```

Comment:
This is probably not what we want for the user field default (i.e. the unix user running the process). At worst this might be a security issue.